src/main/java/seedu/address/logic/commands/DeleteLessonCommand.java

```diff
@@ -38,12 +38,14 @@ public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
         List<Lesson> lastShownList = model.getFilteredLessonList();
 
-        if (targetIndex.getZeroBased() >= lastShownList.size()) {
+        boolean isIndexOutOfBounds = targetIndex.getZeroBased() >= lastShownList.size();
```

Comment:
I did this mainly to make the if statement easier to read.
So the if statement becomes
`if (isIndexOutOfBounds) {...`

instead of that complicated expression.

Should I not do it this way?

Reply:
Also I can't sleep and it's 5am help

Reply:
It's more like I felt the expression wasn't really that complicated.